cs/test/compat/Program.cs

```diff
@@ -106,6 +106,11 @@ static void Write<R>(R reader, OutputStream output, string toProtocol)
                 var writer = new CompactBinaryWriter<OutputStream>(output);
                 Serialize.To(writer, Deserialize<Compat>.From(reader));
             }
+            else if (toProtocol == "compact2")
+            {
+                var writer = new CompactBinaryWriter<OutputStream>(output, 2);
+                Serialize.To(writer, Deserialize<Compat>.From(reader));
+            }
```

Comment:
Yup, I've taken care of this as part of the merge.

https://github.com/Microsoft/bond/commit/5bbe73c7b19f1969483d4743b4fa4b6ab51e8451